docs/UserGuide.md

```diff
@@ -146,6 +146,8 @@ Before continuing, here are some important information you need to know about th
 | **Reopen**   | `reopen INDEX`                                                                                                                                                                    |
 | **View**     | `view INDEX`                                                                                                                                                                  |
 | **Update**   | `update c/COMPANY_INDEX app/APPLICATION_INDEX as/APPLICATION_STATUS`<br> e.g.,`update c/1 app/1 as/OA`                                                                        |
+| **Favourite**| `fav COMPANY_INDEX` |
+| **Unfavourite**| `unfav COMPANY_INDEX` |
```

Comment:
In this context since fav and unfav only uses one INDEX and does not utilise the prefix c/, I feel that fav and unfav formats should be `fav INDEX` and `unfav INDEX`